llvm/lib/Target/AMDGPU/SIInsertWaitcnts.cpp

```diff
@@ -485,6 +485,15 @@ class WaitcntGenerator {
   virtual AMDGPU::Waitcnt getAllZeroWaitcnt(bool IncludeVSCnt) const = 0;
 
   virtual ~WaitcntGenerator() = default;
+
+  // Create a mask value from the initializer list of wait event types.
+  unsigned eventMask(std::initializer_list<WaitEventType> Events) const {
```

Comment:
Doesn't this function need to be constexpr to let you use it in the initializer of static data? (I am not a C++ expert!)

Reply:
Changed the function to a constexpr.